testpmd/hooks/roles/mirror-rh-nfv/tasks/main.yml

```diff
@@ -60,4 +60,27 @@
     - update_status == ['True']
   retries: 60
   delay: 60
+  ignore_errors: true
+
+- name: Retrieve more logs in the case of failure
+  shell: |
+    {{ oc_tool_path }} get mcp
+    echo "=========="
+    {{ oc_tool_path }} get mc
+    echo "=========="
+    {{ oc_tool_path }} describe mcp worker
+    echo "=========="
+    for i in {0..3}; do echo "worker-$i" && oc describe node worker-$i | grep Config; done
```

Comment:
This will fail if the number of workers is note between 0 and 3. We can get the current working nodes using:
```
$ kubectl get nodes --selector=node-role.kubernetes.io/worker -o=custom-columns=NAME:.metadata.name|sed -n '1!p'
worker-0
worker-1
worker-2
worker-3
```
Or from the inventory. Not sure if we are going to still need this now that we have information about OMG https://github.com/dci-labs/dallas-internal-docs/blob/master/must-gather.md 

We need to check if the information that we are trying to get here is already in the must-gather file attached to the job.

Reply:
Nevermind about OMG, describe node is still not implemented. :(
```
$ omg describe node worker-0 
This feature is not yet developed!
```
But it works for the MC stuff
```
$ omg get mc|tail -n3
rendered-worker-14c00588fa3f7c009cc206257e442fd9  c9fce5a27bb71642339085600b98a2e60341a6c2  3.2.0            1h19m
rendered-worker-536b0be40b655d2a23bcc29514bd887c  c9fce5a27bb71642339085600b98a2e60341a6c2  3.2.0            1h19m
rendered-worker-cd9b25e8549e8e03e0f53f4ea7f3ea27  c9fce5a27bb71642339085600b98a2e60341a6c2  3.2.0            2h4m
$ omg get mcp
NAME    CONFIG                                            UPDATED  UPDATING  DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT  DEGRADEDMACHINECOUNT  AGE
worker  rendered-worker-14c00588fa3f7c009cc206257e442fd9  True     False     False     4             4                  4                    0                     2h5m
master  rendered-master-221c26fe086746234d2f2cb325afb159  True     False     False     3             3                  3                    0                     2h
```

Reply:
Hi @betoredhat thank you for the advice, I'm checking must-gather and updating jira issue for today's example-cnf fail